### DISCUSSION TITLE
Making signature required

### DISCUSSION THREAD

**Discussion by NikMesney** [szimek/signature_pad#627] [General]
This might be a simple thing, but I'm just wondering if you can make the signature required, like a required field in a form? Is there a way of checking if the signature is empty and preventing the page from proceeding if so?

**Comment by UziTech** [comment]
You can check `signaturePad.isEmpty()`

**Comment by xitude** [comment]
Im having this issue with users submitting a (or multiple) dots over signatures. Would like a way to avoid it.

**Reply by UziTech** [reply]
You can get the data and check it based on your criteria. For example making sure there is a point group with at least 3 points to make a line.

```js
const data = signaturePad.toData();
if (isNotASignature(data)) {
  alert("This is not a signature!");
  return;
}
```
